internal/sandbox/sandbox.go

```diff
@@ -89,7 +89,7 @@ type Sandbox interface {
 	// The result of the supplied command will be returned in an instance of
 	// RunResult.
 	Run(...string) (*RunResult, error)
-
+	UploadFileToContainer(srcFile string, destFile string) *exec.Cmd
```

Comment:
I don't think this method needs to be in the Sandbox interface.

The `UploadFileToContainer` is only used inside the `Run` implementation below

Reply:
IMO it's a generic function which might be reused in the future, similar to startContainerCmd

Reply:
It is nice to have this code implemented as a function! Since it's only got one usage, how about we leave it as an internal function for now? Then if it is reused/needed in future external code, it can be added to the interface.